src/main/groovy/org/transmartproject/batch/startup/JobStartupDetails.groovy

```diff
@@ -189,18 +195,29 @@ final class JobStartupDetails {
     }
 
     @TypeChecked(TypeCheckingMode.SKIP)
-    @SuppressWarnings('UnusedPrivateMethod')
-    private void validate() throws InvalidParametersFileException {
-        modules.each { it.validate(internalInterface) }
+    private void munge() throws InvalidParametersFileException {
+        checkForEmptyValues()
```

Comment:
This is confusing. `munge()` is now not only munging, but also checking for empty values and for extra parameters (which are actually forms of validation).